### ISSUE TITLE
[Feature Request] Add volume slider

### ISSUE THREAD

**Issue by mslourens** [openopus-org/concertmaster_player#21]
The Spotify SDK supports setting the volume of the played song, so it would be nice if the Concertmaster.app supports this as well by adding a volume slider to the interface.

**Comment by adrianosbr** [comment]
Indeed! Volume adjustment and scrubbing were intentionally left aside because we had too many things to do (for example: a native iOS app is top priority now).
